### DISCUSSION TITLE
Some planet generation experience

### DISCUSSION THREAD

**Discussion by bjperson** [systemed/tilemaker#336] [General]
Hi, 

First of all I would like to thank you for what you do for the community. It's just awesome and so helpful :+1: 

I also wanted to give you some feedback :

```
tilemaker --input ./planet-latest.osm.pbf --output ./planet-latest.mbtiles \
--config ./config-openmaptiles.json --process ./process-openmaptiles.lua --store ./ --threads 0
```

Debian 11
CPU AMD EPYC 7532 32C/64T 2.2 GHz
256 GB RAM
NVMe SSD

Tilemaker master from October 5th (before this day commit)
planet-latest.osm.pbf from October 2nd

Execution time : 7h20min (yes!)
Maximum memory usage : 152G
Store size : 474G

Mbtiles filesize : 91G

I didn't see any problem other than those mentioned in #315 except that Ijsselmeer and the Great Lakes are ok.
Everything looks good :-)

**Comment by systemed** [comment]
That's amazing! Thanks for the report.

**Comment by bjperson** [comment]
I took the time to read a little bit of the different feedbacks and I realize that it might be useful to also share with you my (non-affiliated) supplier and what it cost me.

Scaleway : https://www.scaleway.com/

Offer : Bare Metal (dedicated server)
Machine code : HC-BM2-L
Price : 1.208 €/h vat not included

So, to be wide counting 10h, it costs about 15 euros vat included (if you don't fall asleep in the middle like me).
